src/common/regions/region_db.ts

```diff
@@ -51,6 +52,13 @@ class RegionDB {
     return this.regionsByFips[fips] || null;
   }
 
+  findByFipsCodeStrict(fipsCode: FipsCode): Region {
```

Comment:
👍 pablo suggested `getByFipsCode` so i might go with that.  I'd rather merge for now though and update in a follow on, this will enable a bunch of functionality for the rest